vendor/bat-native-ads/src/bat/ads/internal/processors/behavioral/purchase_intent/purchase_intent_processor.cc

```diff
@@ -237,8 +239,12 @@ void PurchaseIntent::OnTextContentDidChange(
 
   const absl::optional<TabInfo> last_visible_tab =
       TabManager::GetInstance()->GetLastVisibleTab();
-  if (SameDomainOrHost(url,
-                       last_visible_tab ? last_visible_tab->url : GURL())) {
+  if (!last_visible_tab) {
+    return;
+  }
+
+  DCHECK(!redirect_chain.empty());
```

Comment:
should be `DCHECK(last_visible_tab->redirect_chain.back());`

Reply:
Fixed